crates/forge/bin/cmd/inspect.rs

```diff
@@ -592,6 +595,40 @@ fn print_eof(bytecode: Option<CompactBytecode>) -> Result<()> {
     Ok(())
 }
 
+fn check_revive_field(field: &ContractArtifactField) -> Result<bool> {
+    let fields_revive_specific_behavior =
+        [ContractArtifactField::Bytecode, ContractArtifactField::DeployedBytecode];
+
+    let fields_revive_unimplemented_warn = [
+        ContractArtifactField::GasEstimates,
+        ContractArtifactField::StorageLayout,
+        ContractArtifactField::Metadata,
```

Comment:
why this field is not implemented for `metadata`?

Reply:
metadata is solc metadata,
storagelayout is evm layout
GasEstimates is not supported